### PR TITLE
[IMP] sales: update portal.rst 

### PR DESCRIPTION
**Pull request by ray-odoo**
Removed old and probably not needed image of the dashboard showing the Contacts App

### CODE REVIEW

**Comment by AntoineVDV** [comment]
Hi @odoo/sales-doc-review, could you handle this PR?

**Review by meng-odoo — APPROVED** [review]
Hi @ray-odoo, thanks for making this change! Looks good to me, so I'll go ahead and approve.

Hi @StraubCreative, please proceed with technical review. It looks like some code checks were unsuccessful. Thanks!

**Comment by StraubCreative** [comment]
@AntoineVDV or @Feyensv can I have a hand with this? 

I tried doing a `git reset --hard 16.0` with a `git cherry-pick` on top of it and I'm still getting CI errors even though I'm not seeing any build errors locally with `make`. The details of the CI errors also don't make any sense to me, as this branch _should_ be an exact replica of `16.0` with only one commit ahead...

Is this one of those instances where it's better to start fresh with a new feature branch/PR and cherry-pick on top of it?

Thanks in advance for any advice/help 🙏

**Comment by StraubCreative** [comment]
@meng-odoo quick follow up.

Kicking this back to you as there's much to update/clean-up:

- 2nd-person familiar you/your language instead of third-person instructional
- gendered pronouns instead of third-person neutral
- generic image filenames
- images too big, out of date, and/or not in HD
- missing `:alt:` text
- missing `:guilabel:` tags
- 100th character formatting
- typos, grammar, and wording
- high image:copy ratio (which suggests more detailed written content could be added here, although I could be mistaken!)
- etc.

I went ahead and added `:guilabel:` tags where I saw fit, formatted for 100th char, cleaned up a couple typos, and added `:alt:` text but the content needs a decent lift to get it up to Guidelines standards.

Please give it a look and tag @mivu-odoo for CR when you're done. Ty 🙏

**Comment by AntoineVDV** [comment]
@StraubCreative The runbot believed that the branch still had a `master` base because that'd the first base it saw.
image

One way to fix this is to change the target branch of the PR from `16.0` to `master`, then from `master` to `16.0`.

**Comment by StraubCreative** [comment]
`portal.rst` rebuild on [87a5c33](https://github.com/odoo/documentation/pull/3137/commits/87a5c33ded9eaa7ffbf36b433a3199816bedbdac): addressed all outstanding items in [this comment](https://github.com/odoo/documentation/pull/3137#issuecomment-1364814394) and updated screenshots as well using a `14.0` runbot instance.

This rewrite would serve better as a base for all versions of Odoo that we're actively maintaining (v14-16), and we can open future PRs to address any version-specific differences or additional features.

I'll reset the commit history and change the target branch to `14.0` in the next commit.

cc: @meng-odoo @ray-odoo 

Also, wondering--
Wouldn't this doc be better served in **Miscellaneous** section of the documentation, given that it touches so many different app scopes beyond Sales? Wdyt @jcs-odoo ?

File: content/applications/sales/sales/advanced/portal.rst [hunk 1]
```diff
@@ -1,48 +1,64 @@
...
 
-What is Portal access/Who is a portal user?
-============================================
+Portal access is given to users who need the ability to view certain documents or information
+within an Odoo database.
 
-A portal access is given to a user who has the necessity to have access
-to Odoo instance, to view certain documents or information in the
-system.
+Some common use cases for providing portal access include allowing customers to read/view any/all
```
Comment (meng-odoo) [inline content/applications/sales/sales/advanced/portal.rst:8]: The two consecutive slashes were a little confusing to read
```suggestion
Some common use cases for providing portal access include allowing customers to read/view any or all
```

File: content/applications/sales/sales/advanced/portal.rst [hunk 2]
```diff
@@ -1,48 +1,64 @@
...
+   database.
 
-How to give portal access to customers?
-=======================================
+Provide portal access to customers
+----------------------------------
+From the main Odoo dashboard, select the :guilabel:`Contacts` application. If the contact is not
+yet created in the database, click on the :guilabel:`Create` button, enter the details of the
+contact, and then click :guilabel:`Save`. Otherwise, choose an existing contact, and then click on
+the :guilabel:`Action`` drop-down menu located at the top-center of the interface.
```
Comment (meng-odoo) [inline content/applications/sales/sales/advanced/portal.rst:31]: Deleting an extra `
```suggestion
the :guilabel:`Action` drop-down menu located at the top-center of the interface.
```

File: content/applications/sales/sales/advanced/portal.rst [hunk 3]
```diff
@@ -1,48 +1,64 @@
...
-
-.. image:: portal/portal04.png
-    :align: center
-
-An email will be sent to the specified email address, indicating that
-the contact is now a portal user of the respective instance.
+.. note::
+   At any time, portal access can be revoked by navigating to the contact, clicking
+   :menuselection:`Action --> Grant portal access`, and then unselecting the checkbox under the
+   :guilabel:`In Portal` column and clicking `Apply`.
```
Comment (meng-odoo) [inline content/applications/sales/sales/advanced/portal.rst:64]: Adding a guilabel
```suggestion
   :guilabel:`In Portal` column and clicking :guilabel:`Apply`.
```

**Review by meng-odoo — CHANGES_REQUESTED** [review]
Hey @StraubCreative! I just found a couple small typos to fix whenever you push the next commit. Thanks :)

**Comment by StraubCreative** [comment]
Thanks for catching those @meng-odoo 🙏 
All CRs were addressed on [4beb0da](https://github.com/odoo/documentation/pull/3137/commits/4beb0da8edd150ad7c95a4f42be451af6e55c30c). Can you take another look, please?

If all good can move on to final content review with Michelle 🚀

**Review by meng-odoo — APPROVED** [review]
@StraubCreative looks good!
@mivu-odoo this is ready for final review :)

**Comment by StraubCreative** [comment]
Changed target branch to `14.0` so we can FWP the doc rewrite up to `16.0`
Reset + cherry-picked on [04dcbe0](https://github.com/odoo/documentation/pull/3137/commits/04dcbe0e52555ce4cacd434431ac6fa9e2e20391). 
All CI checks pass.

Approving content and moving to @odoo/doc-review 👍

**Review by StraubCreative — APPROVED** [review]

**Review by AntoineVDV — APPROVED** [review]
Delegating so you can merge the fwp PRs.

@robodoo delegate=StraubCreative

**Comment by StraubCreative** [comment]
@robodoo r+

**Pull request closed without merging**